services/QuillLMS/app/models/user.rb

```diff
@@ -233,6 +235,29 @@ def self.valid_email?(email)
     ValidatesEmailFormatOf.validate_email_format(email).nil?
   end
 
+  def require_email_verification
+    create_user_email_verification unless user_email_verification
```

Comment:
Where is `create_user_email_verification` defined?

Reply:
This is Rails magic.  https://guides.rubyonrails.org/association_basics.html#methods-added-by-belongs-to